x11-misc/emwm-utils/emwm-utils-9999.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Utilities for Enhanced Motif Window Manager (EMWM)."
+HOMEPAGE="http://fastestcode.org/emwm.html"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/alx210/mwm-utils.git"
+else
+	SRC_URI="https://fastestcode.org/dl/emwm-utils-src-1.0.tar.xz -> ${P}.tar.xz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+
+BDEPEND="
+	x11-libs/motif
+	>=x11-libs/libX11-1.6.2
+	virtual/libcrypt:=
```

Comment:
libX11 and libcrypt likely belong in DEPEND+RDEPEND.

BDEPEND is for "things we execute". I have no idea if motif installs a binary that gets called during the build process.